config/database.yml.tt

```diff
@@ -0,0 +1,23 @@
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: 5
+
+development:
+  <<: *default
+  host:     <%= ENV.fetch('DB_HOST') { 'localhost' } %>
+  port:     <%= ENV.fetch('DB_PORT') { 5432 } %>
+  username: <%= ENV.fetch('DB_USERNAME') { 'postgres' } %>
+  password: <%= ENV.fetch('DB_PASSWORD') { 'postgres' } %>
+  database: <%= ENV.fetch('DB_NAME') { APP_NAME } %>_development
+
+test:
+  <<: *default
+  host:     <%= ENV.fetch('DB_HOST') { 'localhost' } %>
+  port:     <%= ENV.fetch('DB_PORT') { 5432 } %>
+  username: <%= ENV.fetch('DB_USERNAME') { 'postgres' } %>
+  password: <%= ENV.fetch('DB_PASSWORD') { 'postgres' } %>
+  database: <%= ENV.fetch('DB_NAME') { APP_NAME } %>_test
+
+production:
+  url: <%= ENV.fetch('DATABASE_URL') { '' } %>
```

Comment:
this will only work for heroku afaik

Reply:
Thanks :) 

Andy found a part of the solution in #411 — but the implementation still require some improvements 💭